components/dashboard/src/components/UsageBasedBillingConfig.tsx

```diff
@@ -226,72 +231,43 @@ export default function UsageBasedBillingConfig({ attributionId }: Props) {
-                        <div className="mt-1 text-xl font-semibold flex-grow text-gray-500 dark:text-gray-400">
-                            Pay-as-you-go
-                        </div>
-                        <div className="mt-4 flex space-x-1 text-gray-400 dark:text-gray-500">
-                            <Check className="m-0.5 w-5 h-5" />
-                            <div className="flex flex-col">
-                                <span>
-                                    {currency === "EUR" ? "€" : "$"}0.36 for 10 credits or 1 hour of Standard workspace
-                                    usage, excluding VAT.{" "}
-                                    <a className="gp-link" href="https://www.gitpod.io/pricing#cost-estimator">
-                                        Estimate costs
-                                    </a>
-                                </span>
-                            </div>
-                        </div>
-                        <div className="flex justify-end mt-6 space-x-2">
-                            {stripePortalUrl && (
-                                <a href={stripePortalUrl}>
-                                    <button className="secondary" disabled={!stripePortalUrl}>
-                                        View Past Invoices ↗
-                                    </button>
-                                </a>
-                            )}
-                            <button onClick={() => setShowBillingSetupModal(true)}>Upgrade Plan</button>
-                        </div>
-                    </div>
-                )}
-                {showUpgradeUser && (
+                {(showUpgradeTeam || showUpgradeUser) && (
                     <div className="flex flex-col mt-4 p-4 rounded-xl bg-gray-50 dark:bg-gray-800">
-                        <div className="uppercase text-sm text-gray-400 dark:text-gray-500">Current Plan</div>
-                        <div className="mt-1 text-xl font-semibold flex-grow text-gray-600 dark:text-gray-400">
-                            {usageLimit > 500 ? "Open Source" : "Free"}
-                        </div>
-                        <div className="mt-4 flex space-x-1 text-gray-400 dark:text-gray-500">
-                            <Check className="m-0.5 w-5 h-5 text-orange-500" />
-                            <div className="flex flex-col">
-                                <span className="font-bold text-gray-500 dark:text-gray-400">{usageLimit} credits</span>
-                                <span>
-                                    {usageLimit / 10} hours of Standard workspace usage.{" "}
-                                    <a className="gp-link" href="https://www.gitpod.io/pricing#cost-estimator">
-                                        Estimate costs
-                                    </a>
-                                </span>
-                            </div>
-                        </div>
+                        {usageLimit > 0 && (
```

Comment:
**issue**: Not sure where to comment about this but before upgrading the layout seems broken with missing text information.

| Before upgrading | After upgrading |
|-|-|
| <img width="495" alt="billing-before" src="https://user-images.githubusercontent.com/120486/213675352-01cab0e7-d015-4d28-b241-6c86bcaafbb7.png"> | <img width="495" alt="billing-after" src="https://user-images.githubusercontent.com/120486/213675354-5059c010-116a-4139-9039-9b5ebfd1562a.png"> |